src/components/table/table.js

```diff
@@ -575,6 +578,18 @@ export default {
       type: [Boolean, String],
       default: false
     },
+    selectable: {
+      type: Boolean,
+      default: false
+    },
+    selectMode: {
+      type: String,
+      default: ''
```

Comment:
Should this default to mode `single`

Reply:
No idea.
My personal preference is for the "OS" mode, OTOH I think the "multi" mode might be more intuitive, and "single" of limited use, but that really need thinking as it is a default value, not easily changed after

Reply:
Users can easily change it by setting the `select-mode` prop.

Reply:
I think single mode is more intuitive to the users of the table, which may not know the shift/CMD keyboard controls needed for multi/range

Reply:
the beauty of the 'range' mode is that if the user doesn't use SHIFT, CTRL/CMD keys, it behaves like the 'single' mode.
I see the 'single' mode more like a limited mode when one just want a single selection (for editing a record for ex)

Reply:
Ok, just played with multi, and I can see that being the default.  We do need to name the other mode as `range`, and all modes should be listed in the possible values for `select-mode` (noting which one is default)

Reply:
Still think this should have a default value explicitly set, rather than an empty string to better convey the default mode.

Reply:
an empty string better explain a "default" mode, and is more linked to the code

Reply:
But if people jump to the component reference, and check the props for defaults they see no mode. And most users wouldn't be familiar with the code. They just want to see the options.

Reply:
done